Documentation/Coverlet.MTP.Integration.md

```diff
@@ -28,16 +28,17 @@ dotnet add package coverlet.MTP
 ToDo: Usage details
 
 A sample project file looks like:
+
 ```xml
 <Project Sdk="Microsoft.NET.Sdk">
    <PropertyGroup>
       <TargetFramework>net8.0</TargetFramework>
       <IsPackable>false</IsPackable>
       <IsTestProject>true</IsTestProject>
-      <!-- Enable Microsoft Testing Platform -->
+      <OutputType>Exe</OutputType>
+      <!-- Enable Microsoft Testing Platform - not required for .NET 10 and later -->
```

Comment:
UseMicrosoftTestingPlatformRunner is relevant for .NET 10.

Only TestingPlatformDotnetTestSupport is irrelevant for .NET 10 (and specifically it's about .NET SDK version, not the target framework).

Reply:
Thank you. I relocated comment.